ContactAppCore/Api/EditAreaController.cs

```diff
@@ -39,7 +39,7 @@ public async Task<int> Add([FromForm] string title)
         [HttpPost("AddOffice")]
         public async Task<int> AddOffice([FromForm] string title, [FromForm] int areaId)
         {
-            if (!securityHelper.AllowArea(User, areaId).Result)
+            if (!securityHelper.AllowArea(User, areaId))
```

Comment:
Could you do this on a property field in the class to remove it from the function?